### DISCUSSION TITLE
build error caused by ScreenFragment

### DISCUSSION THREAD

**Discussion by 12ya** [software-mansion/react-native-screens#1289] [Ideas]
/android/src/main/java/com/[...]/ScreenFragment.kt 76 line context: Context,
after deleting comma works good again

**Comment by tboba** [comment]
Hey @12ya, do you know if the problem still exists? I believe you shouldn't spot any problem while building react-native-screens 3.26.0, but let us know by opening the issue if you're still facing the problem.

Thanks!
